test cases/common/260 env externalprogram/meson.build

```diff
@@ -0,0 +1,18 @@
+project('env externalprogram')
+
+py3 = import('python3').find_python()
+
+check_env_exe = find_program('check-env.py', required: true)
+check_env_path = check_env_exe.full_path()
+
+ret = run_command(py3, '-c', 'import os; print(os.environ.get("CUSTOM_ENV_VAR"));',
+  env: {
+    'CUSTOM_ENV_VAR': check_env_exe
+  },
+  check: false
+)
+assert(ret.returncode() == 0, 'failed to run python3: ' + ret.stderr())
+assert(
+  ret.stdout() == '@0@\n'.format(check_env_path),
```

Comment:
Shouldn't this just be `ret.stdout().strip() == check_env_path` instead of using formatting here?

Reply:
I reused the code from the other test, it's a good nitpick.